addons/l10n_sa/models/account_tax.py

```diff
@@ -0,0 +1,9 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    name = fields.Char(translate=True)
```

Comment:
Isn't that already made in l10n_multilang?

Reply:
indeed